session.go

```diff
@@ -836,6 +836,7 @@ func (s *session) PrepareStmt(sql string) (stmtID uint32, paramCount int, fields
 		return
 	}
 
+	s.PrepareTxnCtx(goctx.Background())
```

Comment:
What's the timestamp used when the prepared statement execute?
Why compile a prepare statement need timestamp?
@jackysp @hanfei1991

Reply:
Please see this pr https://github.com/pingcap/tidb/pull/5116 for some details. @tiancaiamao